File: SDKs/youneedabudget.com/1.0.0/python/src/sdk/models/shared/hybridtransactionsresponse.py
from dataclasses import dataclass, field
from typing import List
from dataclasses_json import dataclass_json
from sdk import utils
from . import *


@dataclass_json
@dataclass
class HybridTransactionsResponseData:
    transactions: List[HybridTransaction] = field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('transactions') }})
    

@dataclass_json
@dataclass
class HybridTransactionsResponse:
    data: HybridTransactionsResponseData = field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('data') }})
    

File: SDKs/youneedabudget.com/1.0.0/typescript/src/sdk/models/shared/hybridtransactionsresponse.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { HybridTransaction } from "./hybridtransaction";



export class HybridTransactionsResponseData extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=transactions", elemType: HybridTransaction })
  transactions: HybridTransaction[];
}


export class HybridTransactionsResponse extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=data" })
  data: HybridTransactionsResponseData;
}
